start.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 #
-# Start script for lfp-pay-api
+# Start script for pps-pay-api
 
 APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 if [[ -z "${MESOS_SLAVE_PID}" ]]; then
     source ~/.chs_env/private_env
     source ~/.chs_env/global_env
-    source ~/.chs_env/lfp-pay-api/env
+    source ~/.chs_env/pps-pay-api/env
```

Comment:
I'm not really sure where these environment variables (source) are held - probably needs to be confirmed with Dileep (DevOps) whether this is something we should be changing yet if we're not intending to deploy to other environments this side of Christmas.

It feels to me like we probably shouldn't change this file as part of this PR.
FYI @dgordon-ch

Reply:
Dead on. I was torn on this anyway, so happy to revert.